aws/cmd/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_generate(t *testing.T) {
+	// TODO: Skiped test because I could not find a way to
+	// use 'goimports' inise the Docker image correctly, ideally it's
+	// an issue relted to Modules
```

Comment:
Before merging I'll open an issue to track this.

Reply:
Maybe you can describe the issue some more?

Reply:
Well, what's really "blocking" for me is this, I don't like merging things without tests so I'd try to figure out now why it doesn't work